configure

```diff
@@ -1836,6 +1843,23 @@ samsungtv)
 	_host_os=linux
 	_host_cpu=arm
 	_host_alias=arm-linux-gnueabi
+	;;
+sailfish)
+        _host_os=linux
+	test "x$prefix" = xNONE && prefix=/usr
+	datarootdir='/usr/share/org.scummvm.scummvm'
+	datadir='${datarootdir}/scummvm'
+	docdir='/usr/share/org.scummvm.scummvm/doc/scummvm'
+	if [ -n "$JPEG_PREFIX" ]; then
```

Comment:
Could you add a comment explaining this bit?

Reply:
Done